packages/twenty-front/src/modules/favorites/favorite-folder-picker/scopes/FavoriteFolderPickerScope.tsx

```diff
@@ -0,0 +1,20 @@
+import { FavoriteFolderPickerInstanceContext } from '@/favorites/favorite-folder-picker/states/context/FavoriteFolderPickerInstanceContext';
+import { ReactNode } from 'react';
+
+type FavoriteFolderPickerScopeProps = {
+  children: ReactNode;
+  favoriteFoldersScopeId: string;
+};
+
+export const FavoriteFolderPickerScope = ({
```

Comment:
outdated naming. We don't have "Scopes" anymore. We use ComponentInstanceContext